src/masternodes/govvariables/attributes.cpp

```diff
@@ -91,6 +93,7 @@ const std::map<uint8_t, std::map<std::string, uint8_t>>& ATTRIBUTES::allowedKeys
                 {"loan_payback_fee_pct",TokenKeys::LoanPaybackFeePCT},
                 {"dex_in_fee_pct",      TokenKeys::DexInFeePct},
                 {"dex_out_fee_pct",     TokenKeys::DexOutFeePct},
+                {"dfip2203_disabled",   TokenKeys::DFIP2203Disabled},
```

Comment:
```suggestion
                {"dfip2203",   TokenKeys::DFIP2203},
```

For consistency with `payback_dfi` and `loan_payback`

Reply:
I think this is different, it is more of a blacklist entry, the others are enabled when true, this is disabled when true hence the explicit disabled in the name. Could invert it but this may be more clear this way.

Reply:
You're right, it's clearer this way for a blacklist